encoding/kzg/pointsIO.go

```diff
@@ -217,3 +217,19 @@ func readBytes(reader *bufio.Reader, numBytesToRead uint64) ([]byte, error) {
 	}
 	return buf, nil
 }
+
+func NumberOfPointsInSRSFile(filePath string, pointsSize int64) (uint64, error) {
+	fileStat, errStat := os.Stat(filePath)
+	if errStat != nil {
+		return 0, fmt.Errorf("cannot stat the file %v: %w", filePath, errStat)
+	}
+	fileSizeByte := fileStat.Size()
+	if fileSizeByte%pointsSize != 0 {
+		return 0, fmt.Errorf("corrupted g2 point from the file %v. "+
+			"The size of the file on the provided path has size that is not multiple of %v, which is %v. "+
+			"It indicates there is an incomplete g2 point", filePath, pointsSize, fileSizeByte)
+	}
+	// get the size
```

Comment:
This comment doesn't seem to apply

Reply:
[24e7a7a](https://github.com/Layr-Labs/eigenda/pull/2036/commits/24e7a7a0472685d9b0e97f9ecd92765fcb9c5c85)